src/reducers/cursorDown.ts

```diff
@@ -11,7 +11,7 @@ const cursorDown = (state: State) => {
   // if there is a cursor, get the next logical child, sibling, or uncle
   if (cursor) {
     const { nextThoughts, contextChain } = nextThought(state, cursor)
-    return nextThoughts.length > 0
+    return nextThoughts && nextThoughts.length > 0
```

Comment:
I think you can omit `nextThoughts.length > 0`. A `Path` cannot have a length of 0.